src/types/types.ts

```diff
@@ -273,3 +273,55 @@ export enum LoadingStatus {
   success,
   error,
 }
+
+export interface PairlistsResponse {
```

Comment:
Please move these into a separate file (in this directory).

We started with one types.ts file - but i've since started to move things apart a bit.
This is clearly scoped - so it's a great fit for a dedicated file.